apps/emqx/src/emqx_persistent_session_ds_state.erl

```diff
@@ -37,6 +37,7 @@
 -export([get_subscriptions/1, put_subscription/4, del_subscription/3]).
 
 -export([make_session_iterator/0, session_iterator_next/2]).
+-export([is_expired/1]).
```

Comment:
Please move this function to `emqx_mgmt_api_clients`. `emqx_persistent_session_ds_state` should be kept as a dumb CRUD module that doesn't contain any business logic.

Reply:
done